ext/openssl/ossl_pkey_ec.c

```diff
@@ -596,8 +596,10 @@ static VALUE ossl_ec_group_initialize(int argc, VALUE *argv, VALUE self)
         ossl_raise(rb_eArgError, "wrong number of arguments");
     }
 
-    if (group == NULL)
-        ossl_raise(eEC_GROUP, "");
+#if !defined(LIKELY) && !defined(RB_LIKELY)
+#define LIKELY(x) (x)
+#endif
+    ASSUME(group);
```

Comment:
We really should simply get rid of Ruby 2.3 -- this is the second time this year we hit a 2.3-only issue.

Reply:
Why not update `spec.required_ruby_version`?